SerialNumber: ""

TopicName: "/camera/image"

Width: -1 # -1 default
Height: -1 # -1 default
OffsetX: 0
OffsetY: 0

# Exposure Auto, 0 stands for Off, 1 stands for Once, 2 stands for Continues
ExposureAutoMode: 2
ExposureTime: 5000
AutoExposureTimeLower: 65
AutoExposureTimeUpper: 10000
# Gain Auto, 0 stands for Off, 1 stands for Once, 2 stands for Continues
GainAuto: 2

# Gamma Selector, 0 stands for user, 1 stands for sRGB
GammaSelector: 1
Gamma: 0.7

# Frame Rate
TriggerEnable: false
TriggerSource: 0 # 0, 2
FrameRate: 10

# liv synchoronize
SyncEnable: false
